src/Patterns/Patterns.php

```diff
@@ -0,0 +1,193 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\Patterns;
+
+use Automattic\WooCommerce\Blocks\Domain\Package;
+
+/**
+ * Patterns class
```

Comment:
Perhaps it might be useful to write a brief description of the classes purpose in the comment here also?